multinode/transaction_sender.go

```diff
@@ -160,7 +160,7 @@ func (txSender *TransactionSender[TX, RESULT, CHAIN_ID, RPC]) SendTransaction(ct
 		}
 
 		txSender.wg.Add(1)
-		go txSender.reportSendTxAnomalies(tx, txResultsToReport)
+		go txSender.reportSendTxAnomalies(ctx, tx, txResultsToReport)
```

Comment:
It looks like this goroutine can live past the point of `SendTransaction` returning. Is that true? If so, this context will be cancelled prematurely.